Client/mods/deathmatch/logic/luadefs/CLuaResourceDefs.cpp

```diff
@@ -22,7 +23,7 @@ void CLuaResourceDefs::LoadFunctions()
         {"getResourceFromName", GetResourceFromName},
         {"getResourceRootElement", GetResourceRootElement},
         {"getResourceGUIElement", GetResourceGUIElement},
-        {"getResourceDynamicElementRoot", GetResourceDynamicElementRoot},
+        {"getResourceDynamicElementRoot", ArgumentParser<GetResourceDynamicElementRoot>},
```

Comment:
Shouldn't ArgumentParserWarn be here?

Reply:
Not in this case

Reply:
Can you explain why it doesn't matter here?

Reply:
Because `getResourceDynamicElementRoot` throws an error under one condition, false if invalid parameters were passed (its handled by ArgumentParser)